core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackV4RequestToTemplatePreparationObjectConverter.java

```diff
@@ -170,6 +177,13 @@ public TemplatePreparationObject convert(StackV4Request source) {
                     throw new CloudbreakServiceException("Cannot collect shared service resources from datalake!");
                 }
             }
+            if (environment.getIdBrokerMappingSource() == IdBrokerMappingSource.MOCK && source.getCloudPlatform() == CloudPlatform.AWS) {
```

Comment:
The usage of mock vs user-provided mappings will be eventually driven by some fields in `StackV4Request ` (TODO in a later PR), so I would rather not introduce condition `environment.getIdBrokerMappingSource()` at all. How about always setting mock mappings for now, and later this can be changed to a fallback (default) logic if no user-provided mappings are given in `StackV4Request `?